app/models/auth.py

```diff
@@ -386,5 +391,18 @@ def verify_reset_password_token(cls, token: str) -> PCeen | None:
             return
         return cls.query.get(id)
 
+    @property
+    def demand_made(self) -> int:
+        """Return if the pceen has made a demand for the given club Q season"""
+
+        if (
+            self.clubq_voeux.filter_by(
+                _season_id=models.GlobalSetting.query.filter_by(key="SEASON_NUMBER_CLUB_Q").one().value
+            )
```

Comment:
Là tu construit ta query sans l'exécuter, donc tu n'auras jamais `None` ? Tu veux ajouter un `.one_or_none()` je pense

```suggestion
            self.clubq_voeux.filter_by(
                _season_id=models.GlobalSetting.query.filter_by(key="SEASON_NUMBER_CLUB_Q").one().value
            ).one_or_none()
```

Par ailleurs, j'aurais tendance à récupérer la `season_id` dans une ligne à part au dessus du `if`, ce sera vachement plus lisible 😉

Reply:
Je suis presque sûr que je n'utilise plus cette propriété donc ça va partir sur une suppression. (Le problème étant que je ne peux pas envoyer une variable saison en même temps, et en plus je peux utiliser .exist() pour les query)